proto/aserto/directory/model/v3/model.proto

```diff
@@ -170,10 +94,8 @@ message GetManifestResponse {
 message SetManifestRequest {
     // manifest instance
     oneof msg {
-        // Manifest metadata
-        Metadata metadata                                       = 1;
         // Manifest content
-        Body body                                               = 2;
+        Body body                                               = 1;
```

Comment:
@gertd does it make sense to keep this message as a `oneof` with just one type in anticipation of future extension, or should I just make a regular field in `SetManifestRequest`?

Reply:
Keeping it as a oneof makes sense to me.